lib/core/engine/iteration.js

```diff
@@ -175,14 +175,13 @@ class Iteration {
       }
       return result;
     } catch (e) {
+      log.error(e);
       // In docker we can use a hardcore way to cleanup
       if (options.docker) {
         if (recordVideo) {
           await stop('ffmpeg');
         }
-        await stop(options.browser);
```

Comment:
Is this not needed anymore? It seems not related to logging.

Reply:
sorry too many commits in one.  I think it was never needed because we stop the browser with the driver? we should instead kill the driver if something fails.